packages/client/src/views/SysAdmin/Config/Certificates/Certificates.tsx

```diff
@@ -565,6 +567,10 @@ function mapStateToProps(state: IStoreState) {
   }
 }
 
+function modifyPreviewImage(file: any) {
+  return file
+}
+
```

Comment:
we can remove this extra function

Reply:
> we can remove this extra function

I have updated the function to do what it was intended for.